selfdrive/car/hyundai/carcontroller.py

```diff
@@ -78,15 +81,24 @@ def update(self, c, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
                                    left_lane_warning, right_lane_warning))
 
     if not CS.CP.openpilotLongitudinalControl:
-      if pcm_cancel_cmd:
-        can_sends.append(create_clu11(self.packer, frame, CS.clu11, Buttons.CANCEL))
+      if pcm_cancel_cmd and not self.prev_pcm_cancel:
+        self.cancel_frames = 0
+      if pcm_cancel_cmd and frame % 10 == 0:
+        # cancel by causing a temporary fault in the SCC module, causes a chime
+        # TODO: find a way to make this silent
+        can_sends.extend([create_cancel_command(self.packer)] * 15)
```

Comment:
Then spamming the message reduces our maximum time to cancel. We *usually* cancel in 10 frames without this, but on occasion it jumps up to 30-40 frames. With this, I haven't seen it above 15 on a short test drive